py3status/core.py

```diff
@@ -326,7 +326,7 @@ def setup(self):
             # load and spawn i3status.conf configured modules threads
             self.load_modules(self.py3_modules, user_modules)
 
-    def notify_user(self, msg, level='error', rate_limit=None, key=''):
+    def notify_user(self, msg, level='error', rate_limit=None, module_name=''):
```

Comment:
The reason I did this was just because it's a string in my mind.  If we had a module called None (unlikely) then it would clash.

But I'd keep it as switching to None would cause the line length to go over 79 chars.

It's not important really so if people want to change that's fine,  I have no strong view on it.